### ISSUE TITLE
Kubeflow Dashboard UI - Address A11y Violations

### ISSUE THREAD

**Issue by jenny-s51** [kubeflow/dashboard#64]
While theming the Model Registry UI for consistency with the Central Dashboard, we identified many accessibility issues using the Axe Chrome extension. 

E.g. on the initial screen, there are 66 violations across various severity levels, including critical issues that affect fundamental accessibility. These issues violate WCAG guidelines and need to be resolved for improved screen reader support and keyboard navigation, among other accessibility requirements.

<img width="600" alt="Screenshot 2024-12-02 at 2 03 32 PM" src="https://github.com/user-attachments/assets/e4936826-fb44-4d52-81d9-8babe6919194">

After pulling [these style tokens](https://github.com/kubeflow/kubeflow/blob/4275d99754ac91f6cf5654b03824a73825e9fe55/components/centraldashboard/public/components/main-page.css#L7C1-L13C51) into MR, a violation caused an error in our CI workflow (a color contrast issue), which we are addressing in this [PR](https://github.com/kubeflow/model-registry/pull/596/files#diff-7f270756c8af8f265c067314fab996824c143092c2c836f339d6a66e606476eaR60). However, the majority of the violations remain in the Kubeflow Dashboard UI.

The MR team is aware of these issues, and we’re prioritizing fixes to improve accessibility as we continue driving consistency between these UIs. We encourage the Kubeflow team to also prioritize accessibility in future UI changes cc @ederign

**Comment by ederign** [comment]
@jenny-s51 thank you so much for opening this. As example when there is a violation in MR here is how we break it via GHA:

<img width="1235" alt="Screenshot 2024-12-04 at 10 12 00 AM" src="https://github.com/user-attachments/assets/12870928-85e7-4f01-83bc-79a8773bd5e2">

We will do the same for Notebooks 2.0. (@paulovmr can you sync with @jenny-s51 and MR team to apply the same action there?)

@tariq-hasan I saw that you are planning to work on 'angular rewrite' of Dashboard. What do you think about adding same time of constraints there to make sure that we address eventual a11y issues?

**Comment by ederign** [comment]
Meanwhile, if someone is interested in community on working to fix accessibility issues on Kubeflow Central dashboard, just let me know. Those are usually low hanging fruits and excelent first time contributions!

**Comment by tariq-hasan** [comment]
> @jenny-s51 thank you so much for opening this. As example when there is a violation in MR here is how we break it via GHA:
> 
> <img alt="Screenshot 2024-12-04 at 10 12 00 AM" width="1235" src="https://private-user-images.githubusercontent.com/531351/392449585-12870928-85e7-4f01-83bc-79a8773bd5e2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM5NTQzMjEsIm5iZiI6MTczMzk1NDAyMSwicGF0aCI6Ii81MzEzNTEvMzkyNDQ5NTg1LTEyODcwOTI4LTg1ZTctNGYwMS04M2JjLTc5YTg3NzNiZDVlMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxMVQyMTUzNDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hZDE1MDNkYTAzZjQ5YjU2YTc0MWNmNWJkNGRhNDFiMDQ4OTA1N2JmZTM5ZDM0MGEwZjM3NTg5OGNiYTk5ZjdhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.v7XgZagKnHuyMH2SYSVj5mAANtbCRYWAXq-QekUfFVs">
> We will do the same for Notebooks 2.0. (@paulovmr can you sync with @jenny-s51 and MR team to apply the same action there?)
> 
> @tariq-hasan I saw that you are planning to work on 'angular rewrite' of Dashboard. What do you think about adding same time of constraints there to make sure that we address eventual a11y issues?

Sure. Will keep it in consideration @ederign.

**Comment by orfeas-k** [comment]
> Those are usually low hanging fruits and excelent first time contributions!

We could definitely add such label to this issue too so we can guide new contributors looking into contributing to kf's dashboard.
